native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -137,8 +142,24 @@ DecoderState::next(const double *probs,
               float score = 0.0;
               std::vector<std::string> ngram;
               ngram = ext_scorer_->make_ngram(prefix_to_score);
+
+              // hot_boost == 1.0 == no boost at all
+              float hot_boost = 1.0;
+              if (!hot_words_.empty()) {
+                // increase prob of prefix for every word
+                // that matches a word in the hot-words list
+                for (std::string word : ngram) {
```

Comment:
have you measured perf impact with scorers?

Reply:
perf as in WER? or perf as in latency?

Reply:
latency

Reply:
I have not measured the latency effects yet, no. 

Are there any TC jobs that do this, or should I profile locally? What do you recommend?

Reply:
Unfortunately, you'd have to do it locally. Using `perf` should be quite easy.